docs/RELEASE.md

```diff
@@ -0,0 +1,54 @@
+# Release process and check offs (client only)
+______
+since there is no server component at this time
+
+## Prepare release
+1. [ ]  User testing on mockups (design)
+1. [ ]  go/nogo on individual features (engineering)
+1. [ ]  create release tag for the selected features
+1. [ ]  all automated tests pass
+1. [ ]  all internationalization tests pass
```

Comment:
Not sure how this differentiates from the automated tests

Reply:
i am not sure how you are testing internationalization. But, while there can be automated tests ensuring that there are internationalized strings for every key, I don't think that there can be automated tests to ensure the **quality** of those strings. Presumably the translation team has some kind of additional quality control. This captures those additional tests.

Reply:
@sapte1 you can also chime in here 😄

Reply:
You are correct! Instead of "internationalization tests", can you modify to "LQA tests" (Localization Quality Assurance) ? Not yet sure what those LQA tests will entail so will be a placeholder for now.